project_timesheet_analytic_update/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```

Comment:
Codacy found an issue: [Missing docstring in public package (D104)](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6119964)